binding/jms/src/test/java/io/tracee/binding/jms/TraceeMessageListenerTest.java

```diff
@@ -47,22 +47,31 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
 
 	@Test
 	public void testDecodesFromMessage() throws Exception {
-		encodedContext.put("contextFromMessage","yes");
+		encodedContext.put(INVOCATION_ID_KEY, "yes");
 		unit.intercept(invocationContext);
-		assertThat(backend.getValuesBeforeLastClear(), hasEntry("contextFromMessage", "yes"));
+		assertThat(backend.getValuesBeforeLastClear(), hasEntry(INVOCATION_ID_KEY, "yes"));
+		verify(mdbLike).onMessage(message);
 	}
 
 	@Test
 	public void testCleansUpContextAfterProcessing() throws Exception {
-		encodedContext.put("contextFromMessage","yes");
+		encodedContext.put("contextFromMessage", "yes");
 		unit.intercept(invocationContext);
 		assertThat(backend.isEmpty(), is(true));
+		verify(mdbLike).onMessage(message);
+	}
+
+	@Test
+	public void testInvocationIdGeneratedIfAbsent() throws Exception {
+		unit.intercept(invocationContext);
```

Comment:
@SvenBunge As you may see, I also changed the preparation step in #testDecodesFromMessage  
from 
    
     encodedContext.put("contextFromMessage","yes");
     unit.intercept(invocationContext);
     assertThat(backend.getValuesBeforeLastClear(), hasEntry("contextFromMessage", "yes"));

to 

    encodedContext.put(INVOCATION_ID_KEY, "yes");
    unit.intercept(invocationContext);
    assertThat(backend.getValuesBeforeLastClear(), hasEntry(INVOCATION_ID_KEY, "yes"));

So the original assertion in MdbLike is still covered(and with verify), and makes #testDecodesFromMessage a contrast to #testInvocationIdGeneratedIfAbsent. 

With encodedContext.put("contextFromMessage","yes") in #testDecodesFromMessage is OK, but looks like a little bit noise. 

This is the reason why I made these changes. If you still think we should recover the MdbLike assertion with this context, please leave a message here, I'll change it later :-)